About.html

```diff
@@ -1,122 +1,166 @@
+				<div class="hamburguer"></div>
+				<div class="hamburguer"></div>
+				<div class="hamburguer"></div>
+			</div>
+			<div id="hiden-header">
+				<div id="social-media">
+					<ul id="social-list">
+						<li class="social"><i class="fa-brands fa-facebook-f"></i></li>
+						<li class="social"><i class="fa-brands fa-twitter"></i></li>
+						<li class="social">English</li>
+						<li class="social">My Page</li>
+						<li class="social">Log Out</li>
+					</ul>
+				</div>
+				<nav id="nav-bar">
+					<a href="index.html">
+						<div id="logo-container">
+							<img src="Images/torii (1).png" alt="logo" class="logo" />
+							<div id="logo-subContainer">
+								<h1>Japanese</h1>
+								<h3>Culture Festival 2023</h3>
+							</div>
+						</div>
+					</a>
+					<ul id="nav-list">
+						<li class="nav-sections"><a href="About.html">About</a></li>
+						<li class="nav-sections">Program</li>
+						<li class="nav-sections">Join</li>
+						<li class="nav-sections">Sponsor</li>
+						<li class="nav-sections">News</li>
+						<li class="nav-sections">Campaign</li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main>
+			<section id="title-about">
+				<div id="title-first">
+					<div class="title-div">
+						<h2>こんにちは世界</h2>
+						<h1>Japanese Culture Festival 2023</h1>
+						<hr class="hr-bar" id="hr-hide">
+					</div>
+				</div>
+				<div id="title-second">
+					<p
+						>The Japan Culture Festival focus on share the beautiful and ancient culture of the
+						Asian region with a lot of activities. This year, we hope to bring more people around
+						the world.<span class="text-hide2"> According to the curriculum guidelines of the Ministry of Education, Culture, Sports, Science and Technology, cultural festivals are part of special activities and are defined as "events which aim to use the results of everyday learning to heighten motivation". <br>The cultural festivals are parts of regular lessons in elementary schools, junior high schools, and high schools, so the students are obligated to attend for graduation. In universities, the cultural festivals are placed as extracurricular activity, so attendance is not required.</span></p
+					>
+					<p
+						>Please contact us per Email for any further questions about Japan Culture Festival
+						2023!</p
+					>
+					<p>japancf2023@culture.org</p>
+				</div>
+			</section>
+			<section id="logo-section">
+				<div id="logo-explain">
+					<div class="title-div">
+						<h1>Japan Culture Festival 2023 Logo</h1>
+						<hr class="hr-bar">
+					</div>
+					<p
+						>The Logo of Japan Culture Festival 2023 was decided through the logo competition from
+						2.February to 3.March.<span class="text-hide2">After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.<br>After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</span></p
```

Comment:
You did well with the `about` page and it looks really close to the design, however, please implement these few changes listed below

1. The top section of this page should have a `background-image` as it is shown in the design which will give it a less simpler look compared to what we have with your design, Kindly implement this for a better UI 

Your design : 
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/fc8bde5c-101a-4be9-b74f-5f8007b6ce77)

Design given :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/b3724ee9-25c0-47f2-8a1e-f57541975ab8)

2. In the mobile view, the partners section background color is really darker compared to what we have in the given design. the design color used was `#414246` 

Your design :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/4fdbfc4a-6352-4dc3-92f5-f91d4cd61908)

Design given :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/aeb3238c-7a17-4e43-a633-b4e11dba5ad5)